tests/storage/datavolume.go

```diff
@@ -917,7 +917,7 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 				}
 			}
 
-			DescribeTable("should resolve DataVolume sourceRef", func(createDataSourceFunc func() *cdiv1.DataSource) {
+			DescribeTable("should resolve DataVolume sourceRef", decorators.Conformance, func(createDataSourceFunc func() *cdiv1.DataSource) {
```

Comment:
Isn't this already tested in CDI?
If so, can it be converted to a a unit test, ensuring that KubeVirt creates a properly-defined DataVolume?

Reply:
No, resolving is done in operators that embed datavolumes in their CRs (kubevirt), for security reasons.
I think converting to a unit test is possible, regardless, but definitely not in this PR

Reply:
Maybe the unit test already exists? If this test should be demoted or deleted, it should not be promoted to `Conformance`.

Reply:
https://github.com/kubevirt/kubevirt/issues/13598

Reply:
> Maybe the unit test already exists? If this test should be demoted or deleted, it should not be promoted to `Conformance`.

Agree, let's leave it for now, it requires a careful closer look. As long as it runs in CI we're gating against the security issue.

Reply:
> Agree, let's leave it for now, it requires a careful closer look. As long as it runs in CI we're gating against the security issue.

Ack. Please keep it out of `Conformance` until the closer look.